.travis.yml

```diff
@@ -1,23 +0,0 @@
-language: python
-python:
-- 3.6
-cache: pip
-install:
-- pip install coveralls tox-travis
-before_script:
-- npm install -g gulp-cli
-- npm config set package-lock false
-script:
-- tox
-- python setup.py bdist_wheel
-after_success:
-- coveralls
-deploy:
-  provider: releases
-  skip_cleanup: true
-  api_key:
-    secure: e24xlCO9zGNwFqicknFg/HAqui5duz2aQM2ruPFDpPbmw4rMD9ABYMq9DUqu/o4bXQMtCqH32fgaKnz6j4LYoMHEedPYpPuluszqXj2coHUoE+p93w/AYrGFiGfMukUtcW8xBxJnLJi4HOovKdSKqaravb7BCCtc/OjfcrPOn+g=
-  file_glob: true
-  file: 'dist/*.whl'
-  on:
-      tags: true
```

Comment:
It looks like this PR currently removes the building of wheels for new releases. Can we migrate that to GitHub Actions, too?

Reply:
Yes, I will create a GitHub Action to build wheels for new releases.
I plan to create a deploy.yml to replace the travis.yml file.